### DISCUSSION TITLE
挑戰張家凱的文章資訊量判斷AI

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1471] [General]
第一次發文，很好奇文章資訊量判斷AI的能力與背後原理，所以我要來做個小實驗。
老師PPT裡說到AI是依據：
2學期5000篇的「github課程討論資料」並利用「BERT模型」AI判斷文章「原創性與資料量」

5000篇的課程討論資料量夠嗎？
又是怎麼利用BERT模型？
怎麼定義「原創性與資料量」？
到底準確度如何?
我想來實驗一下
以下是一些實驗素材跟遠則選擇此素材原因

維基百科介紹台灣：不原創但富含資訊
舊的貼文：跟以往資料完全相同
佛經：跟以往資料(應該)完全不同，語序、語詞極偏門
指定Chatgpt生成原創貼文
指定Chatgpt生成原創貼文(挪威文)：測試言語種類的影響
指定Chatgpt生成沒原創貼文
指定Chatgpt生成沒原創貼文(挪威文)
幹話產生器

**Comment by roma0323** [comment]
維基百科介紹台灣：
臺灣（俗字寫作台灣；達悟語：Ilaod），西方世界亦稱福爾摩沙（葡萄牙語：Formosa），是位於東亞、太平洋西北側的島嶼，地處琉球群島與菲律賓群島之間，西隔臺灣海峽與中國大陸相望，海峽距離約130公里，周圍海域從3點鐘方向以順時鐘排序分別為太平洋（菲律賓海）、巴士海峽、南海、臺灣海峽、東海。面積約3.6萬平方公里，在當前全球各島嶼面積排名中位居第38（或39），島上約七成面積之地形為山地與丘陵，平原主要集中於西部沿海，地形海拔變化大，最高點3952公尺。

**Comment by roma0323** [comment]
舊的貼文：
WDI 研發的特技機器人共經歷四代，最原型是稱為「磚塊」（the brick）的機器人，能控制重心，並能控制空中姿態及自動旋轉。
透過程式控制，在繩上搖擺時先屈膝加速，達一定角度後「鬆手」藉慣性擺向空中，同時伸展身體，並由「磚塊」做完美轉體，最後躺平落在網上，完成一整套「蜘蛛人」在大樓間穿梭的動作。
WDI 再把兩套棍子機器人組合做成類人樣，便成為原型機，在此基礎上最佳化等，最終成為 Stuntronics 類人型特技機器人。從棍子機器人到 Stuntronics，WDI 團隊接受《Wired》採訪時省略許多未講，並沒有多說團隊的努力，只說如何使機器人更像真的蜘蛛人，以及確保安全性。
Stuntronics 外殼是樹脂，3D 列印製作，即使減速網破掉，WDI 團隊也表示樹脂外殼能保護內部昂貴精密的主機板、處理器及線路。
https://technews.tw/2021/10/16/walt-disney-imagineering-stuntronics/

**Comment by roma0323** [comment]
佛經：
觀自在菩薩。行深般若波羅蜜多時。照見五蘊皆空。度一切苦厄。舍利子。色不異空。空不異色。色即是空。空即是色。受想行識。亦復如是。舍利子。是諸法空相。不生不滅。不垢不淨。不增不減。是故空中無色。無受想行識。無眼耳鼻舌身意。無色聲香味觸法。無眼界。乃至無意識界。無無明。亦無無明盡。乃至無老死。亦無老死盡。無苦集滅道。無智亦無得。以無所得故。菩提薩埵。依般若波羅蜜多故。心無罣礙。無罣礙故。無有恐怖。遠離顛倒夢想。究竟涅槃。三世諸佛。依般若波羅蜜多故。得阿耨多羅三藐三菩提。故知般若波羅蜜多。是大神咒。是大明咒。是無上咒。是無等等咒。能除一切苦。真實不虛。故說般若波羅蜜多咒。即說咒曰。
揭諦揭諦　波羅揭諦　波羅僧揭諦　菩提薩婆訶

**Comment by roma0323** [comment]
Chatgpt 原創貼文
標題：「探索新興科技：區塊鏈與物聯網的無限潛能」
區塊鏈和物聯網是當今科技領域最為熱門的話題之一。區塊鏈作為一種去中心化的數據儲存和交易技術，正在改變我們的金融、供應鏈和資訊安全等領域。物聯網則通過連接物理設備和傳感器，實現了設備之間的通信和數據共享。

這兩項技術的結合帶來了無限的潛能。例如，在供應鏈管理中，區塊鏈可以提供透明且不可篡改的交易紀錄，同時物聯網可以實時監測和追蹤物品的運送過程。這樣的結合可以大幅提高供應鏈的效率和可靠性。

此外，區塊鏈和物聯網的結合還可以應用於智能城市領域。通過連接各種設備和系統，例如交通信號燈、智能垃圾桶和能源管理系統，我們可以實現城市基礎設施的智能化管理和優化。同時，區塊鏈的安全性和去中心化特性可以保護數據的隱私和安全。

區塊鏈和物聯網的結合還可以應用於能源管理、智能合約、醫療保健等領域。這些新興技術的應用將改變我們的生活方式和工作方式，創造更加便利和智能的社會。

總結而言，區塊鏈和物聯網的結合將帶來無限的創新和潛能。這兩項技術的結合可以改變供應鏈管理、智能城市、能源管理和醫療保健等領域，為我們的生活帶來更多便利和智能化的選項。讓我們拭目以待，見證這個充滿潛力的未來！

**Comment by roma0323** [comment]
Chatgpt 沒原創貼文
嗨大家！今天天氣真好！我想分享一下我今天吃了一個超好吃的漢堡，真的是超級好吃！然後我還去看了一場電影，笑得肚子都痛了！真的是超好笑的！晚上還和朋友去唱歌，唱得超high！好開心啊！最後回家的路上，我看到了一隻超可愛的小貓咪，好像想帶回家養！總之今天過得太棒了，真希望每天都可以這麼開心！希望大家也有一個美好的一天！加油！

**Comment by roma0323** [comment]
Chatgpt原創有資訊(挪威文)
Hei alle sammen! I dag skal vi utforske det vakre norske språket og oppdage noen interessante fakta om Norge. Norsk er det offisielle språket i Norge og snakkes av rundt 5 millioner mennesker over hele landet.

Norsk er et nordgermansk språk som har to hovedformer: bokmål og nynorsk. Bokmål er den mest brukte formen og er basert på dansk-norsk skriftspråk. Nynorsk er en alternativ form som er mer basert på norske dialekter. Begge formene er likeverdige og offisielle.

Norsk har også noen spesielle fonetiske trekk, som for eksempel bruken av tonem. Dette betyr at ordets tonehøyde kan endre betydningen. For eksempel kan "hunden" og "hunden" bety henholdsvis "hunden" og "å hate".

Norge er kjent for sin storslåtte natur, med fjorder, fjell og vakre landskap. Landet er også kjent for sin rike kulturarv, inkludert sin folklore og litteratur. Verk som "Peer Gynt" av Henrik Ibsen og eventyrene til Asbjørnsen og Moe er eksempler på norsk litteratur som har blitt berømt over hele verden.

I tillegg til norsk, snakker mange nordmenn også engelsk som et andre språk, og du vil finne det enkelt å kommunisere med lokalbefolkningen når du besøker Norge.

Det er bare noen få interessante fakta om norsk språk og kultur. Norge er et fascinerende land med mye å utforske. Så hvorfor ikke lære noen norske fraser og planlegge en tur til dette fantastiske landet?

**Comment by roma0323** [comment]
Chatgpt無原創資訊(挪威文)

Hei alle sammen! Håper alle har det bra i dag. Sommeren er her, og det betyr sol, varme og moro. Jeg vil bare dele noen av mine tanker og ønsker for denne fantastiske årstiden.

Sommeren er virkelig den beste tiden å nyte utendørsaktiviteter. Enten det er å dra på piknik i parken, grille med venner eller ta en forfriskende dukkert i sjøen, er det noe for enhver smak. Det er også en flott tid for å reise og utforske nye steder. Kanskje du har planer om en spennende tur eller bare ønsker å slappe av på en vakker strand.

Jeg elsker også alle de deilige sommermatene som er tilgjengelige. Fra ferske bær og frukt til grillet mat og iskrem, er det så mange smaker å glede seg over. Og la oss ikke glemme de fargerike og forfriskende drinkene som kan nytes på en varm sommerdag.

Uansett hva dine planer er for sommeren, håper jeg at du får en fantastisk tid fylt med glede, latter og gode minner. Ta vare på deg selv og dine kjære, og nyt alt det sommeren har å tilby.

Ha en strålende sommer, og kos deg masse!

**Comment by roma0323** [comment]
幹話產生器：
諸葛亮講過一句話，鞠躬盡瘁，死而後已。帶著這句話，我們還要更加慎重的審視這個問題：我這幾天一直在思索這個問題，馬克思有一句名言，一切節省，歸根到底都歸結為時間的節省。這讓我思索了許久，既然如此，而這些並不是完全重要，更加重要的問題是，別林斯基有說過一句話，好的書籍是最貴重的珍寶。這讓我深深地想到，現在，解決先有雞還是先有蛋的問題，是非常非常重要的。 所以，郭沫若講過一句話，形成天才的決定因素應該是勤奮。這不禁令我深思。為什麼是這樣呢？先有雞還是先有蛋真的是很值得探究，一般來講，我們都必須務必慎重的考慮考慮。我們一般認為，抓住了問題的關鍵，其他一切則會迎刃而解。你真的了解先有雞還是先有蛋嗎？雪萊曾經告訴世人，過去屬於死神，未來屬於你自己。這果然是一句至理名言。

**Comment by Aaron0810** [comment]
結果看起來你所有文章都成功了是嗎? 我感覺說不定外文成功率比較高，不過這要取決於老師放的判定機器人就是了，你讓我也想試試看po一些外文文章了。

**Comment by chia-kai-chang** [comment]
謝謝 @roma0323 測試，由於是測試文章，不是發問問題或分享具有個人觀點的資訊，所以不計點

如果對模型訓練有興趣，也歡迎加入 BorgLab ，一起來寫論文
